src/Illuminate/Support/Str.php

```diff
@@ -83,8 +83,12 @@ public static function of($string)
      * @param  string  $search
      * @return string
      */
-    public static function after($subject, $search)
+    public static function after($subject, $search, $ignoreCase = false)
     {
+        if ($ignoreCase) {
+            return $search === '' ? $subject : array_reverse(preg_split("/$search/i", $subject, 2))[0];
```

Comment:
@bmckay959 would it make sense to use preg_split here now that we are using regular expressions?

```suggestion
            $search = preg_qoute($search);
            return $search === '' ? $subject : array_reverse(preg_split("/$search/i", $subject, 2))[0];
```

Reply:
I think you comment meant, "use preg_quote here..."?

If so, I think the answer is yes. I'm not super familiar with regex and didn't even know preg_quote existed, I'll try to build out a few test cases that need to have something escaped today and add it.

Thanks!